internal/defaults/defaults.go

```diff
@@ -17,60 +17,140 @@ package defaults
 	HubbleRelayServerCertValidityDuration = 3 * 365 * 24 * time.Hour
-	HubbleRelayServerCertSecretName       = "hubble-relay-server-certs"
-
-	HubbleRelayClientCertGenerate         = false
-	HubbleRelayClientCertCommonName       = "*.hubble-relay.cilium.io"
+	// HubbleRelayServerCertSecretName is the Kubernetes Secret in which the
+	// Hubble Relay server certificate is written to.
+	HubbleRelayServerCertSecretName = "hubble-relay-server-certs" //#nosec
+
+	// HubbleRelayClientCertGenerate can be set to true to generate and store a
+	// Hubble Relay client TLS certificate (used for the mTLS handshake with
+	// the Hubble servers).
+	HubbleRelayClientCertGenerate = false
+	// HubbleRelayClientCertCommonName is the Hubble Relay client x509
+	// certificate CN value.
+	HubbleRelayClientCertCommonName = "*.hubble-relay.cilium.io"
+	// HubbleRelayClientCertValidityDuration represent how much time the Hubble
+	// Relay client certificate generated by certgen is valid.
 	HubbleRelayClientCertValidityDuration = 3 * 365 * 24 * time.Hour
-	HubbleRelayClientCertSecretName       = "hubble-relay-client-certs"
-
-	CiliumNamespace = "kube-system"
-
-	ClustermeshApiserverServerCertGenerate         = false
-	ClustermeshApiserverServerCertCommonName       = "clustermesh-apiserver.cilium.io"
-	ClustermeshApiserverServerCertValidityDuration = 3 * 365 * 24 * time.Hour
-	ClustermeshApiserverServerCertSecretName       = "clustermesh-apiserver-server-cert"
-
-	ClustermeshApiserverAdminCertGenerate         = false
-	ClustermeshApiserverAdminCertCommonName       = "root"
-	ClustermeshApiserverAdminCertValidityDuration = 3 * 365 * 24 * time.Hour
-	ClustermeshApiserverAdminCertSecretName       = "clustermesh-apiserver-admin-cert"
-
-	ClustermeshApiserverClientCertGenerate         = false
-	ClustermeshApiserverClientCertCommonName       = "externalworkload"
-	ClustermeshApiserverClientCertValidityDuration = 3 * 365 * 24 * time.Hour
-	ClustermeshApiserverClientCertSecretName       = "clustermesh-apiserver-client-cert"
-
-	ClustermeshApiserverRemoteCertGenerate         = false
-	ClustermeshApiserverRemoteCertCommonName       = "remote"
-	ClustermeshApiserverRemoteCertValidityDuration = 3 * 365 * 24 * time.Hour
-	ClustermeshApiserverRemoteCertSecretName       = "clustermesh-apiserver-remote-cert"
-
-	K8sRequestTimeout = 60 * time.Second
+	// HubbleRelayClientCertSecretName is the Kubernetes Secret in which the
+	// Hubble Relay client certificate is written to.
+	HubbleRelayClientCertSecretName = "hubble-relay-client-certs" //#nosec
+
+	// ClustermeshApiserverServerCertGenerate can be set to true to generate
+	// and store a new Clustermesh API server TLS certificate.
+	ClustermeshApiserverServerCertGenerate = false //nolint:revive
+	// ClustermeshApiserverServerCertCommonName is the Clustermesh API server
+	// x509 certificate CN value (also used as DNS SAN).
+	ClustermeshApiserverServerCertCommonName = "clustermesh-apiserver.cilium.io" //nolint:revive
```

Comment:
I assume the linter complained about the overly long variable name?

Reply:
My bad I forgot to remove them.